pages/common/cs-resolve.md

```diff
@@ -0,0 +1,28 @@
+# cs resolve
+
+> Resolve lists the transitive dependencies of one or more other dependencies.
+> More information: <https://get-coursier.io/docs/cli-resolve>.
+
+- Resolve lists of transitive dependencies of two dependencies:
+
+`cs resolve {{groupId1}}:{{artifactId1}}:{{artifactVersion1}} {{groupId2}}:{{artifactId2}}:{{artifactVersion2}}`
+
+- Resolve lists of transitive dependencies of a package by the dependency tree:
+
+`cs resolve --tree {{groupId}}:{{artifactId}}:{{artifactVersion}}`
+
+- Resolve dependency tree in a reverse order (from a dependency to its dependencies):
+
+`cs resolve --reverse-tree {{groupId}}:{{artifactId}}:{{artifactVersion}}`
+
+- Print all the libraries that depends on a particoular library:
+
+`cs resolve {{groupId}}:{{artifactId}}:{{artifactVersion}} --what-depends-on {{searchedGroupId}}:{{searchedArtifactId}}`
+
```

Comment:
```suggestion
- Print all the libraries that depends on a specific library:

`cs resolve {{group_id}}:{{artifact_id}}:{{artifact_version}} --what-depends-on {{searched_group_id}}:{{searched_artifact_id}}`
```

The same issue as above.
Optional fix: broken description: `specific` expected.